docs/configuration/janitor.md

```
@@ -0,0 +1,259 @@
+## TTL-Based CR Cleanup
+
+Completed CRs are automatically deleted after the TTL expires.
+
+```yaml
+janitor:
+  ttl:
+    enabled: true
+    defaultTTL: "336h"
+```
+
+#### enabled
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)